torch/csrc/jit/script/init.cpp

```diff
@@ -193,14 +194,67 @@ struct VISIBILITY_HIDDEN ConstantPythonTupleValue : public PythonValue {
   }
 };
 
+
```

Comment:
I doubt it is a good idea to add more SugaredValue types simply to support this one module. This needs a more thorough explanation of what is going on, and why these changes are necessary.